lightly_studio/src/lightly_studio/resolvers/image_resolver/get_all_by_collection_id.py

```diff
@@ -24,6 +25,49 @@ class GetAllSamplesByCollectionIdResult(BaseModel):
     samples: Sequence[ImageTable]
     total_count: int
     next_cursor: int | None = None
+    similarity_scores: Sequence[float] | None = None
+
+
+def _get_distance_expression(
```

Comment:
We should merge this with the video part and refactor the code. Will make a follow up issue for that.